qiita_pet/templates/portals_edit.html

```diff
@@ -0,0 +1,96 @@
+{% extends sitebase.html%}
+
+{%block head%}
+<link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
+<script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript">
+function check_submit(action) {
+  error = false;
+  var data = oTable.$('input').serialize();
```

Comment:
Not sure how it works (and I can't see the interface so far) but is it possible to add an event listener to the dataTable object so you can catch when the user choses something and then you make the button clickable? This way you can avoid this check here...